content/guides/component-testing/custom-mount-vue.md

```diff
@@ -201,29 +201,16 @@ component**!
 Custom `cy.mount` commands to the rescue! You may find the JSX syntax to be more
 straightforward.
 
-You'll also need to replicate the plugin setup steps from the Vuetify docs for
-everything to compile.
-
 <code-group>
 
 <code-block label="JavaScript" active>
 
 ```js
-import Vuetify from 'vuetify/lib'
 import { VApp } from 'vuetify'
 import { mount } from 'cypress/vue'
 import { h } from 'vue'
 
-// We recommend that you pull this out
-// into a constants file that you share with
-// your main.js file.
```

Comment:
I think we need to keep the main docs as the Vue 3 + latest Vuetify info, I think most people would expect them to match the latest version.